frontend/components/ticket/AttachmentsTab.vue

```diff
@@ -67,6 +79,10 @@
         this.files = files
 
         this.submit()
+      },
+      show_delete(uploader_id) {
```

Comment:
In de file wordt camel casing en snake casing door elkaar gebruikt, miss ff veranderen